web-app/src/store/user/mutations.ts

```diff
@@ -0,0 +1,10 @@
+import { MutationTree } from 'vuex';
+import { UserStateInterface } from './state';
+
+const mutation: MutationTree<UserStateInterface> = {
```

Comment:
Add `logOut()` mutation for logout to restore default values

Reply:
Added